### ISSUE TITLE
(high) bookmarklet buttons aren't working as intended

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-easy-checks#163]
Location: 
Example: 
https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/required-fields/ -see attached
https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/image-alt/ -see attached

![easy-checks-alt-text](https://github.com/w3c/wai-easy-checks/assets/18597908/7f6f2059-a11d-499d-b7ff-cd53844d728e)
![easy-checks-required-fields](https://github.com/w3c/wai-easy-checks/assets/18597908/48e30aaa-0077-40e0-a8ed-de7f178183d8)
Current wording:  needs more on using these bookmarklets.  It may be user error on my part but not sure what to do with them

Suggested revision:

Rationale:

**Comment by iadawn** [comment]
The second screen shot seems to be the result of you having pressed the 'Check' button repeatedly on the page. I can change it so that it doesn't keep adding the error messages.

Coupled with the close button this may help this situation.

However, I am not clear on what you mean about the current wording not being clear. Which wording?

**Comment by lakeen** [comment]
First, I should say I love this latest revision of Easy Checks. It's so much cleaner and much more readable. Being a developer, I went straight to trying to break stuff :)

When I navigated to the individual pages I did find the "Check" buttons a bit confusing.  I like, for example, the check required fields behavior.  The only thing I would add is a close X to the Find Out More modal.  Also, I think managing the error messages will help pages like alt text.  I actually did not notice the change highlighting the W3c logo at the top of the page.

I should have been clearer about the wording I was having trouble with.  Specifically:
"Drag the [Check required fields] link to your browser bookmarks."  
1. both buttons are identical so am I dragging the first button or the one in the instruction.
2. when I tried I was not successful so that's the confusing part.  At first I thought I'm in Edge and not Chrome so maybe it's that.  So I tried Chrome and I got the same O with a diagonal line through it indicating I couldn't do what you're asking.  Does that make sense?
